__tests__/app.test.js

```diff
@@ -251,3 +251,23 @@ describe.only('POST /api/articles/:article_id/comments', () => {
         })
     })
 
+    describe.only('GET /api/users', () => {
+        test('Returns array of objects with correct properties', () => {
+            return request(app)
+            .get('/api/users')
+            .then(({body}) => {
+                expect(body.users).toHaveLength(4);
+                body.users.forEach((users) => {
+                    expect(users).toHaveProperty('username')
+                    expect(users).toHaveProperty('name')
+                    expect(users).toHaveProperty('avatar_url')
+                    
+                })
+            })
+        })
```

Comment:
Your test on line 35 of this file already test for ALL bad paths, this test is unnecessary